packages/tasks/src/model-libraries.ts

```diff
@@ -114,6 +114,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path_extension:"npz"`,
 	},
+	birefnet: {
+		prettyLabel: "BiRefNet",
+		repoName: "BiRefNet",
+		repoUrl: "https://github.com/ZhengPeng7/BiRefNet",
+		docsUrl: "https://www.birefnet.top",
```

Comment:
```suggestion
```

This is not properly a documentation URL so better to remove it I think. Link to `repoUrl` is good :+1: